modules/functional-test/src/main/java/com/intuit/wasabi/tests/service/RapidExperimentationTest.java

```diff
@@ -0,0 +1,328 @@
+package com.intuit.wasabi.tests.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import org.apache.http.HttpStatus;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.intuit.wasabi.tests.library.TestBase;
+import com.intuit.wasabi.tests.library.util.Constants;
+import com.intuit.wasabi.tests.model.Application;
+import com.intuit.wasabi.tests.model.Assignment;
+import com.intuit.wasabi.tests.model.Bucket;
+import com.intuit.wasabi.tests.model.Experiment;
+import com.intuit.wasabi.tests.model.Page;
+import com.intuit.wasabi.tests.model.User;
+import com.intuit.wasabi.tests.model.factory.AssignmentFactory;
+import com.intuit.wasabi.tests.model.factory.BucketFactory;
+import com.intuit.wasabi.tests.model.factory.ExperimentFactory;
+
+/*******************************************************************************
+ * Copyright 2017 Intuit
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *******************************************************************************/
+public class RapidExperimentationTest extends TestBase {
+    /**
+     * This test class contains functional tests that check the rapid experimentation functionality
+     * 
+     * Created on 05/23/2017
+     */
+    private static final int RAPID_EXP_MAX_USERS = 5; // max cap on number of users
```

Comment:
i think this would be better as i dont want to overload the config file and making it confusing..since this constant is only for rapid experimentation test...i feel its better we use the constants